go.mod

```diff
@@ -14,6 +14,8 @@ require (
 	github.com/tshihad/tftags v0.0.10
 )
 
+replace github.com/HewlettPackard/hpegl-vmaas-cmp-go-sdk => ../hpegl-vmaas-cmp-go-sdk
```

Comment:
Pl remove after testing

Reply:
Will-do once the PR has landed in hpegl-vmaas-cmp-go-sdk